### DISCUSSION TITLE
new gotek AT32F435 ''COMPATIBILITY''

### DISCUSSION THREAD

**Discussion by EnsoniqSamplers** [keirf/flashfloppy#780] [General]
Hello to everyone, i am a new member here and have installed the old Gotek in several of my vintage sampler but i would like to replace them with the new Gotek (2022) because of the bigger screen and better cpu!

Could anyone tell me please if the new gotek AT32F435 is compatible with the following samplers, Ensoniq Mirage and Sequential Prophet 2002?

thnx in advance

**Comment by keirf** [comment]
It will be no less compatible than the old Goteks running FlashFloppy. But you will have more reliability in certain operations, especially formatting a disk.

**Reply by EnsoniqSamplers** [reply]
Hello keirf, it worked is more stable than previews versions and the bigger screen is an improvement, the actual software is flashfloppy 3.38 and i would like to do some modifications like the oled screen to stay always ON also the 2 buttons to be used for entering inside the folder and going out of the folder as make no sense to have buttons and encoder doing the same function.
so what is the best way to do this ?
thanks again for your help

**Reply by keirf** [reply]
You can make these changes in the FF.CFG configuration file, https://github.com/keirf/FlashFloppy/wiki/FF.CFG-Configuration-File

There is also an example file shipped with FlashFloppy under `examples/`.

Specifically, you want the following in FF.CFG for your big OLED display and the behaviour you want:
```
display-type = oled-128x64-rotate
display-off-secs = 0
twobutton-action = rotary
```

That last option will mean you can leave a folder at any time by pushing button 1. There is no specific button action for entering a folder, as that is simply the same action as selecting an image.
